localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
+e_black_dragonflight_adj;Obsidian;obsidienne;pechschwarz;;obsidiana;;;;;;;;;x
+e_red_dragonflight;Red Dragonflight;Vol draconique rouge;Roter Drachenschwarm;;Vuelo Rojo;;;;;;;;;x
+e_red_dragonflight_adj;Vermillion;vermillon;zinnober;;bermellón;;;;;;;;;x
+e_blue_dragonflight;Blue Dragonflight;Vol draconique bleu;Blauer Drachenschwarm;;Vuelo Azul;;;;;;;;;x
+e_blue_dragonflight_adj;Azure;azur;azur;;azur;;;;;;;;;x
+e_bronze_dragonflight;Bronze Dragonflight;Vol draconique de bronze;Bronzener Drachenschwarm;;Vuelo Bronce;;;;;;;;;x
+e_bronze_dragonflight_adj;Bronze;bronze;bronzen;;bronce;;;;;;;;;x
+d_silverwing_sentinels;Silverwing Sentinels;Sentinelles d'Aile-argent;Schildwachen der Silverwing;;Centinelas Ala de Plata;;;;;;;;;x
+d_silverwing_sentinels_adj;Sentinel;Sentinelle;schildwache;;Centinela;;;;;;;;;x
+d_cenarion_circle;Cenarion Circle;Cercle cénarien;Zirkel des Cenarius;;Círculo Cenarion;;;;;;;;;x
+d_cenarion_circle_adj;Cenarion;Cénarienne;cenarisch;;Cenarion;;;;;;;;;x
+d_wardens;Wardens;Gardiennes;Wächterinnen;;Celadoras;;;;;;;;;x
+d_wardens_adj;Warden;Gardienne;der Wächterinnen;;Celadora;;;;;;;;;x
+k_wildhammer;Grim Batol;Grim Batol;;;Grim Batol;;;;;;;;;x
 k_wildhammer_adj;Grim Batol;Grim Batol;;;;;;;;;;;;x
-d_bloodsail_buccaneers;Bloodsail Buccaneers;Boucaniers de la Voile Sanglante;Blutsegel-Bukanier;;;;;;;;;;;x
-d_northsea_freebooters;Northsea Freebooters;Flibustiers des mers du Nord;Nordsee-Freibeuter;;;;;;;;;;;x
+d_bloodsail_buccaneers;Bloodsail Buccaneers;Voile sanglante;Blutsegelbukaniere;;Bucaneros Velasangre;;;;;;;;;x
+d_northsea_freebooters;Northsea Freebooters;Flibustiers des mers du Nord;Nordmeerfreibeuter;;Filibusteros del Mar del Norte;;;;;;;;;x
 d_spirestone;Spirestone;Pierre-du-pic;Felsspitze;;;;;;;;;;;x
-d_smoulderthorn;Smoulderthorn;Br�leronce;Gluthauer;;;;;;;;;;;x
-k_cenarion_circle;Cenarion Circle;Cercle c�narien;Zirkel des Cenarius;;;;;;;;;;;x
-k_cenarion_circle_adj;Cenarion;C�narienne;cenarisch;;;;;;;;;;;x
-k_exodar;Exodar;Exodar;;;;;;;;;;;;x
+d_smoulderthorn;Smolderthorn;Brûleronce;Gluthauer;;;;;;;;;;;x
+k_cenarion_circle;Cenarion Circle;Cercle cénarien;Zirkel des Cenarius;;Círculo Cenarion;;;;;;;;;x
+k_cenarion_circle_adj;Cenarion;Cénarienne;cenarisch;;Cenarion;;;;;;;;;x
+k_exodar;Exodar;Exodar;;;Exodar;;;;;;;;;x
 k_exodar_adj;Exodarian;Exodarienne;;;;;;;;;;;;x
-e_pantheon;Titan Pantheon;Panth�on;Pantheon der Titanen;;;;;;;;;;;x
+e_pantheon;Pantheon;Panthéon;Pantheon;;Panteón;;;;;;;;;x
 e_herald_cthun;C'Thun Empire;Empire de C'Thun;C'Thuns Imperium;;;;;;;;;;;x
 e_herald_nzoth;N'Zoth Empire;Empire de N'Zoth;N'Zoths Imperium;;;;;;;;;;;x
 e_herald_yoggsaron;Yogg-Saron Empire;Empire de Yogg-Saron;Yogg-Sarons Imperium;;;;;;;;;;;x
 e_herald_yshaarj;Y'Shaarj Empire;Empire de Y'Shaarj;Y'Shaarjs Imperium;;;;;;;;;;;x
-e_valarjar;Halls of Valour;Salles des Valeureux;Hallen der Ehre;;;;;;;;;;;x
-d_roughnecks;Roughnecks;Baroudeurs;Raubhafen;;;;;;;;;;;x
-d_roughnecks_adj;Roughnecks;Baroudeuse;Raubhafen;;;;;;;;;;;x

```

Comment:
Did I really translate these? Or were the english ones changed after I wrote those couple german ones? Maybe I was tired when I worked on these... Good job improving it.

Reply:
I don't know who did these lines, I just pulled them up in WoWhead and changed the locale to German and went accordingly. So thank Blizzard's translations, I just copied it. :P

Reply:
That was meant as a compliment, as the last sentence said. If it was done by me earlier, shame on me ;)